tests/integration/components/modifiers/enable-tooltip-test.ts

```diff
@@ -0,0 +1,154 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { Placement } from '@upfluence/oss-components/test-support/custom-assertions/tooltip';
+
+const PLACEMENTS: Placement[] = [undefined, 'top', 'bottom', 'left', 'right'];
+
+module('Integration | Component | modifiers/enable-tooltip', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function () {
+    this.title = 'Title';
+  });
+
+  test("it doesn't renders when the title is empty", async function (assert) {
+    this.title = '';
+    await render(hbs`
+      <div class="test-container" style="height: 20px; width: 40px" 
+           {{enable-tooltip title=""}}>
+      </div>
+    `);
+    await assert.tooltip('.test-container').doesNotExist();
+  });
+
+  test('it renders the correct title', async function (assert) {
+    this.title = 'title';
+    await render(hbs`
+      <div class="test-container" style="height: 20px; width: 40px"
+           {{enable-tooltip title=this.title}}>
+      </div>
+    `);
+    await assert.tooltip('.test-container').hasTitle(this.title);
+  });
+
+  module('icon attribute', () => {
+    test('it renders the correct icon', async function (assert) {
+      this.icon = 'far fa-wine-glass-alt';
+      await render(hbs`
+      <div class="test-container" style="height: 20px; width: 40px"
+           {{enable-tooltip title=this.title icon=this.icon}}>
+      </div>
+    `);
```

Comment:
since the render call is 5 lines long, and as you call it everywhere in the file, I suggest you create a `renderTooltip()` function to make things clearer.

```
async function renderTooltip() {
   await render(hbs`
      <div class="test-container" style="height: 20px; width: 40px"
           {{enable-tooltip title=this.title icon=this.icon}}>
      </div>
    `);
}
```

used w/ `await renderTooltip()`
Also watch out, the indentation of the render above is wrong